src/simple_lottery/init.rs

```diff
@@ -0,0 +1,94 @@
+//! Parameter derivation for simple lottery.
+
+use super::params::Params;
+use super::setup::Setup;
+
+use std::f64::consts::LN_2;
+
+/// Calculates Setup parameters.
+pub fn make_setup(params: &Params) -> Setup {
```

Comment:
Just a question: are we planning to expose this function to the public API (i.e., allow the users to call it)? If so, we might consider adding more specific comments on its usage - something like the `prove/verify` functions in @rrtoledo 's PRs. Here is a suggested template for the docs: 
```
 /// Adds two numbers together.
///
/// # Arguments
///
/// * `a` - The first integer to add.
/// * `b` - The second integer to add.
///
/// # Returns
///
/// An integer that is the sum of `a` and `b`.
///
/// # Examples
///
/// ```
/// let result = add(2, 3);
/// assert_eq!(result, 5);
/// ```
```
My comment is identical to the comments in @rrtoledo 's PRs.

Reply:
Already done in https://github.com/cardano-scaling/alba/pull/66